### DISCUSSION TITLE
OrganizationページのContributorsの表示順を、Orgに紐付いた記事のContributions順にしてほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#634] [要望・アイデア・不具合]
Organizationページ（例: https://qiita.com/organizations/dosukoi-juku ）におけるContributorsの表示順について、現状だとOrg関係なくContributionsの多い順に左から並んでいるようです。
Orgに貢献した人ほど左に並んでほしいので、Orgに紐付いた記事のみのContributions順に変更していただけると嬉しいです。

**Comment by tanaka-asami415** [comment]
@uhooi 
リクエストありがとうございます。確かにです！
時期は先になるかもしれませんが、対応させていただきます！
